### PR TITLE
Mesh Generation with Spheres

### PR DESCRIPTION
**Pull request by arthbous**
@maximilianmetti take a look at aux/test_gmsh (actually it should be mshr since that's what I use)

### CODE REVIEW

**Comment by arthbous** [comment]
All done @maximilianmetti

**Comment by arthbous** [comment]
Change the folder to ``benchmarks/battery``

**Comment by arthbous** [comment]
@maximilianmetti The benchmark for the sphere is done. Now we can just play with it.

File: benchmarks/battery/battery.cpp [hunk 1]
```diff
@@ -0,0 +1,680 @@
...
+        printf("\nSuccessfully solved the system below desired residual in %d steps!\n\n", newton_iteration);
+      else {
+        printf("\nDid not converge in %d Newton iterations at t=%e...\n", max_newton_iters,t);
+        printf("\tcurrent relative residual is %e > %e\n\n", relative_residual, nonlinear_tol);
+      }
+
+      // compute local entropy and refine mesh
+      printf("Computing electric field for refinement\n");
+      unsigned int num_refines;
+      num_refines = check_electric_field (
```
Comment (maximilianmetti) [inline benchmarks/battery/battery.cpp]: this function does not exist...

**Pull request closed without merging**